specs/AppNotifications/BadgeNotificationSpec/BadgeNotifications-spec.md

```diff
@@ -0,0 +1,125 @@
+# Badge notifications in Windows App SDK
+
+<!-- Put an index here - see how dynamic dependencies doc do it -->
+
+- [1. Background](#background)
+- [2. Limitations of the Windows SDK APIs](#limitations-of-the-windows-sdk-apis)
+- [3. API proposal](#api-proposal)
+- [4. Remarks](#remarks)
+- [5. IDL details](#idl-details)
+
+## Background
+
+Badges are a type of notifications indicating a summary or status information specific to an app. It is represented either by a number or a glyph sitting over the top-right corner of the taskbar app icon.
+
+Here is an example of a numeric badge posted by Microsoft To-Do List:
+
+![Example of a numeric badge in Microsoft To-Do List](NumericBadge.png)
+
+Another example of a badge, in this case featuring a new message glyph posted by Outlook:
+
+![Example of a busy status glyph badge in Microsoft Teams](GlyphBadge.png)
+
+## Limitations of the Windows SDK APIs
+
+Badge notification Windows SDK APIs have three main limitations:
+
+- They doesn't support Win32 apps, whether the app is packaged or not.
+
+- The Badge Windows SDK APIs expect an XML payload that contains the information about the badge to pin, even though the XML is a single line of content. There are no APIs where developers can directly send the number or specify the glyph type.
+
+- As of today, Badge APIs support a limited set of glyphs. If the app doesn't find any suitable glyph to convey their status, there is no option to pin a custom overlay.
```

Comment:
Should also mention that we won't be supporting Badge updates via WNS.

Reply:
And I don't think that concern will be addressed initially right?

Reply:
I hesitated about mentioning cloud badges, but yeah, we can add this in the remarks section.